servers/src/main/java/tachyon/worker/block/BlockStore.java

```diff
@@ -176,6 +176,13 @@ TempBlockMeta createBlockMeta(long userId, long blockId, BlockStoreLocation loca
   void accessBlock(long userId, long blockId) throws IOException;
 
   /**
+   * Get the block metadata manager.
+   *
+   * @return block metadata manager
+   */
+  BlockMetadataManager getBlockMetadataManager();
```

Comment:
I didn't find a good way to do it without exposing the metadata manager. There is only a `BlockStore`  in the `BlockDataManager`. Do you want to expose the two functions in BlockStore so BlockDataManager can expose them too?